dashboards/flink/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+sample_dashboards:
+  -
+    category: flink
```

Comment:
```suggestion
    category: Flink
```

Can you capitalize Flink here and in the other category field?

Reply:
updated